### ISSUE TITLE
Loses channel balance

### ISSUE THREAD

**Issue by marcmicalizzi** [malensek/3RVX#33]
I have 7.1 surround with the balance for the speaker levels set differently for every speaker, sometimes applying settings and on startup the balance of every level is set to the same number.

**Comment by malensek** [comment]
Hey @marcmicalizzi , are you referring to these levels:
<img width="221" alt="levels" src="https://cloud.githubusercontent.com/assets/2635718/9499343/3c0c3924-4bdc-11e5-99a9-6552748cb688.png">
So far the behavior of 3RVX seemed similar to the slider in the notification area, but if this is where you're seeing the problem then I'll keep experimenting to see if I can reproduce the problem. Thanks a lot!

**Comment by marcmicalizzi** [comment]
Yes, that's the level balances i'm referring to. In 3RVX 2 the problem happened every windows bootup after 3RVX started. In 3RVX 3 I only managed to reproduce it once so far however, and it was inconsistent. I changed some settings, clicked apply, it was fine, changed some more, click apply, then the volume changed. I wish I could provide more concise steps to reproduce.

My base volume settings on Balance are 12/12/12/20/28/28/16/16 in the unlikely event that changes anything.
